frontend/static/js/services/ARequestService.js

```diff
@@ -138,7 +138,7 @@ class ARequestService {
 			method: 'DELETE',
 			headers: {
 				'X-CSRFToken': getCookie('csrftoken'),
-				'Content-Type': 'application/json',
```

Comment:
I dont think this is a good idea, the rest of the deletions will be json wont they?

Reply:
In general, deleteRequest doesnt really take a body, just an id of the resource we want to delete. how about an overload?